ts/parts/Pointer.ts

```diff
@@ -834,7 +856,7 @@ Highcharts.Pointer.prototype = {
             chart.hoverPoint = hoverPoint;
 
             // Draw tooltip if necessary
-            if (tooltip) {
+            if (tooltip && (!chart.polar || hoverPoint.isInsidePane)) {
```

Comment:
Instead of checking `hoverPoint.isInsidePane`, how about `fireEvent(this, 'afterGetHoverData', hoverPoint);` can remove hoverPoint? Something like this:

```js
let e = { hoverPoint };
fireEvent(this, 'afterGetHoverData', e);
hoverPoints = e.hoverPoint;
```

And `hoverPoint` will be undefined when outside the pane.